tags:: paper, rejected
link:: [LINK](https://jitc.bmj.com/content/9/3/e001717)
authors:: Wilkins A et al
n:: 53
dose-cgy:: NA
time-post-rt:: post-treatment
disease-site:: rectal cancer
pub-date:: 2021
tissue-pre-rt:: biopsy
tissue-post-rt:: resection specimens
fractionation:: NA
deg-comparison:: paired pre/post same patient

- We generated GEPs for 53 patients from biopsies taken prior to preoperative radiotherapy.
- TCD was used to assess rectal tumor response to neoadjuvant RT/CRT and ΔTCD was subjected to k-means clustering to classify patients into different response categories.
	- DEG was performed by response category, which was determined by delta TCD.
	- no sequencing done on post RT resection specimen
- Differential gene expression analysis was performed using statistical analysis of **microarrays, pathway enrichment analysis and immune cell type analysis using single sample gene set enrichment analysis**. Immunohistochemistry was performed to validate specific results.
	- Also used microarrays
- The results were validated using 220 pretreatment samples from publicly available datasets at metalevel of pathway and survival analyses.
- Longitudinal (treatment-induced) significant changes in gene expression were evaluated in paired samples of good and poor responders in two separate SAM analyses.[20](https://jitc.bmj.com/content/9/3/e001717#ref-20) For pathway analysis, the Molecular Signature Database (MSigDB)’s hallmarks gene sets[26](https://jitc.bmj.com/content/9/3/e001717#ref-26) and *hypeR R* package[21](https://jitc.bmj.com/content/9/3/e001717#ref-21) was used to describe the biological pathways represented by differentially expressed genes.
	- DEG based on response category
	-